example/demo-gen/demo/ICU4XFixedDecimalFormatter.mjs

```diff
@@ -0,0 +1,50 @@
+import { ICU4XDataProvider } from "../ICU4XDataProvider.mjs"
+import { ICU4XFixedDecimal } from "../ICU4XFixedDecimal.mjs"
+import { ICU4XFixedDecimalFormatter } from "../ICU4XFixedDecimalFormatter.mjs"
+import { ICU4XFixedDecimalFormatterOptions } from "../ICU4XFixedDecimalFormatterOptions.mjs"
+import { ICU4XLocale } from "../ICU4XLocale.mjs"
+
+export function formatWrite(name, grouping_strategy, some_other_config, v) {
+	var terminusArgs = arguments;
+	return (function (...args) { return this.formatWrite(...args) }).apply(
+        ICU4XFixedDecimalFormatter.tryNew.apply(
+        null,
+        [
+            ICU4XLocale.new_.apply(
+                null,
+                [
+                    terminusArgs[0]
+                ]
+            ),
+            ICU4XDataProvider.newStatic.apply(
+                null,
+                [
+                ]
+            ),
+            (function (...args) {
+            	let out = new ICU4XFixedDecimalFormatterOptions();
+            	
+            	out.groupingStrategy = args[0];
+            	
+            	out.someOtherConfig = args[1];
+            	
+            	return out;
+            }).apply(
+                null,
+                [
+                    terminusArgs[1],
+                    terminusArgs[2]
+                ]
+            )
+        ]
+    ),
+        [
+            ICU4XFixedDecimal.new_.apply(
+                null,
+                [
+                    terminusArgs[3]
```

Comment:
Suggestion: If you have enough information to know that "it is the third argument", you probably also know the variable name.

Reply:
Unfortunately, I had to axe including parameter names in anything but the typescript definition. Right now we don't check for collision with parameter names, but I will add this to the issues list.